README.md

```diff
@@ -205,6 +205,16 @@ To use the rake tasks, `require puppet-strings/rake_tasks` in your `Rakefile`:
 require 'puppet-strings/rake_tasks'
 ```
 
+If you need a more configurable version of the rake task, you can use `puppet-strings/rake_tasks/generate`.
+
+```ruby
+PuppetStrings::RakeTasks::Generate.new(:documentation) do |task|
+  task.paths = ['site/roles','site/profiles','modules/internal']
+  task.ignore = ['/vendor/','/example/']
+  task.options = {} # disables the strings.json output
+end
+```
+
```

Comment:
Sorry, but I don't understand how this works - the names are very similar. It says "If you need a more configurable version of the rake task, you can use `puppet-strings/rake_tasks/generate`" as opposed to what? Also, does this alter the meaning of the normal generate ? (I did not quite have the time to read up in detail how the pieces fit together). I think a bit more explanation of how it is supposed to work / be used would be helpful.